frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -139,8 +145,13 @@ const ClusterAlerts = withDashboardResources(
     }
 
     if (alertsLoaded && !_.isEmpty(alerts)) {
+      const filteredAlerts = isClusterDashboard
+        ? alerts
+        : alerts.filter((alert) => _.get(alert?.labels, [labelFilterKey]) === labelFilterValue);
```

Comment:
Prefer optional chaining, which is type safe

```suggestion
        : alerts.filter((alert) => alert?.labels?.[labelFilterKey] === labelFilterValue);
```

Reply:
@spadgett ok, good to know that can be used with the [] syntax as well.